packages/simple-form/src/modules/simpleForm/sagas.js

```diff
@@ -30,12 +34,18 @@ export function* initialize() {
 
 export function* submit() {
   const values = yield select(getFormValues(FORM_ID))
-  yield put(externalEvents.fireExternalEvent('onSubmit', {...values}))
+  yield put(externalEvents.fireExternalEvent('onSubmit', {values}))
 }
 
 export function* cancel() {
   const values = yield select(getFormValues(FORM_ID))
-  yield put(externalEvents.fireExternalEvent('onCancel', {...values}))
+  yield put(externalEvents.fireExternalEvent('onCancel', {values}))
+}
+
+export function* change() {
+  const values = yield select(getFormValues(FORM_ID))
```

Comment:
in general, selector arguments should be passed as follows, shouldn't they?

`yield select(getFormValues, FORM_ID))`

(for testability, since `getFormValues(FORM_ID) !== getFormValues(FORM_ID)`)

Reply:
true for typical selector function. but i think this does not work in this case because getFormValues is not meant to be used in sagas as selector?